src/slack/dispatch.js

```diff
@@ -118,7 +123,7 @@ const getAnythingElse = (record) => {
   return anythingElseObject;
 };
 
-const getVolunteers = (volunteers) => {
+const getVolunteers = (volunteers, taskCounts) => {
```

Comment:
In #42, `getVolunteers` and all of there message formatting helper have been moved to `src/slack/message/index.js`.